src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        String additionalZero = "";
+        if (number < 10) {
+            additionalZero += 0;
+        }
+        return "Ball: " + additionalZero + number + " " + color;
+    }
```

Comment:
All of that should not be a part of the `Ball` class logic, `toString()` should just give us a string representation of that class, without any counters, `if`s or anything basically, if you want to do this in a pretty way, just add some more logic to the main method.